src/components/ShippingDetails/__tests__/__snapshots__/ShippingDetails.test.js.snap

```diff
@@ -0,0 +1,63 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`COMPONENT - ShippingDetails should render correctly ShippingDetails component 1`] = `
+.c0 {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+.c1 {
+  -webkit-box-flex: 1;
+  -webkit-flex-grow: 1;
+  -ms-flex-positive: 1;
+  flex-grow: 1;
+  padding: 0 1rem;
+}
+
+.c2 {
+  font-weight: bold;
+}
+
+<div
+  className="c0"
+>
+  <svg
```

Comment:
Snapshot of icon is rather irrelevant for this test. You can mock it in test file: `jest.mock('../../Icons', () => ({ Shipping: 'Shipping' }))`